server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/QuotaGetMethodContract.scala

```diff
@@ -0,0 +1,813 @@
+         |    "methodResponses": [
+         |        [
+         |            "Quota/get",
+         |            {
+         |                "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+         |                "notFound": [ ],
+         |                "state": "${INSTANCE.value}",
+         |                "list": [
+         |                    {
+         |                        "used": 1,
+         |                        "name": "account:count:Mail",
+         |                        "id": "fbb433deb379b7ba87e2df898887056a0fa645839ee091d13e4da99e52b4c4f6",
+         |                        "dataTypes": [
+         |                            "Mail"
+         |                        ],
+         |                        "limit": 100,
+         |                        "resourceType": "count",
+         |                        "scope": "account"
+         |                    },
+         |                    {
+         |                        "used": 85,
+         |                        "name": "account:octets:Storage",
+         |                        "id": "fbb433deb379b7ba87e2df898887056a0fa645839ee091d13e4da99e52b4c4f6",
+         |                        "dataTypes": [
+         |                            "Storage"
+         |                        ],
+         |                        "limit": 900,
+         |                        "resourceType": "octets",
+         |                        "scope": "account"
+         |                    }
+         |                ]
+         |            },
+         |            "c1"
+         |        ]
+         |    ]
+         |}
+         |""".stripMargin)
+  }
+
+
+  @Test
+  def quotaGetShouldFailWhenWrongAccountId(): Unit = {
+    val request =
+      s"""{
+         |  "using": [
+         |    "urn:ietf:params:jmap:core",
+         |    "urn:ietf:params:jmap:quota"],
+         |  "methodCalls": [[
+         |    "Quota/get",
+         |    {
+         |      "accountId": "unknownAccountId",
+         |      "ids": null
+         |    },
+         |    "c1"]]
+         |}""".stripMargin
+
+    val response = `given`
+      .body(request)
+    .when
+      .post
+    .`then`
+      .log().ifValidationFails()
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .extract
+      .body
+      .asString
+
+    assertThatJson(response).isEqualTo(
+      s"""{
+         |  "sessionState": "${SESSION_STATE.value}",
+         |  "methodResponses": [
+         |    ["error", {
+         |      "type": "accountNotFound"
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin)
+  }
+
+  @Test
+  def quotaGetShouldFailWhenOmittingOneCapability(): Unit = {
+    val request =
+      s"""{
+         |  "using": [
+         |    "urn:ietf:params:jmap:core"],
+         |  "methodCalls": [[
+         |    "Quota/get",
+         |    {
+         |      "accountId": "unknownAccountId",
```

Comment:
In `quotaGetShouldFailWhenOmittingOneCapability` I guess you should use the valid accountId